### ISSUE TITLE
"Unknown Type" error on table in BlockList

### ISSUE THREAD

**Issue by twhit-mrwebapps** [dosco/graphjin#220]
### What version of GraphJin are you using? `graphjin version`
v0.16.50

### Have you tried reproducing the issue with the latest release?
On latest
Go Version: go1.16.3

### What is the hardware spec (RAM, OS)?
Debian 10
16GB
i7-2600k
Postgresql

### Steps to reproduce the issue (config used to run GraphJin).
Added specific tables to the BlockList in the Config.

`cfg.Blocklist = []string {
		"password",
		"goose_db_version",
		"policy_actions",
		"salt",
		"secret",
	}`

**DB Table**
`CREATE TYPE POLICYITEMTYPE AS ENUM('REST_ACTION', 'SCHEMA', 'SPA_MENU_ITEM', 'SPA_ACTION');

CREATE TABLE policy_actions (
	id CHARACTER(26) NOT NULL,
	policy_item_type POLICYITEMTYPE,
	policy_item_name VARCHAR(50) UNIQUE NOT NULL,
	is_writable BOOLEAN DEFAULT false NOT NULL,
	active BOOLEAN DEFAULT true NOT NULL,
	created_at TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp,
	updated_at TIMESTAMP WITH TIME ZONE DEFAULT NULL,
	deleted_at TIMESTAMP WITH TIME ZONE DEFAULT NULL,
	PRIMARY KEY (id)
)`

### Expected behaviour and actual result.
Running go run main.go is exiting with the following error:

2021/05/09 17:24:49 graphql: Unknown type "policy_actionsOutput". (line 0, column 0)graphql: Unknown type "policy_actionsOutput". (line 0, column 0)
exit status 1

Is it possible this issue is coming from the "TYPE" I've created in the table above? I'm not sure how this error is occurring. Commenting this out from the Blocklist works fine. I can even access it without issue or errors.

Appreciate any feedback.

**Comment by dosco** [comment]
I'll take a look

**Comment by dosco** [comment]
Hmm, I'm not able to recreate this issue. I tested with the `webshop` example app that comes built in. I added the above table and custom table to the database and queried it with the web ui. I also tried after adding `policy_actions` to the blocklist.

<img width="1037" alt="Screen Shot 2021-05-09 at 7 31 31 PM" src="https://user-images.githubusercontent.com/832235/117598801-7d07eb80-b116-11eb-9fff-609e960a932f.png">
<img width="1106" alt="Screen Shot 2021-05-09 at 7 33 11 PM" src="https://user-images.githubusercontent.com/832235/117598837-901abb80-b116-11eb-9ef2-79f8537d1b4c.png">
